src/tests/ftest/util/daos_utils.py

```diff
@@ -621,3 +636,116 @@ def pool_list_cont(self, pool, svc, sys_name=None, env=None):
         self.sub_command_class.sub_command_class.sys_name.value = sys_name
         self.env = env
         return self._get_result()
+
+    def pool_set_attr(self, pool, attr, value, svc, env=None):
```

Comment:
Using `env` in these 'helper' methods to set `self.env` should be removed.  Since these methods are part of the object, the setting of the env should only be needed once:
```
    daos = DaosCommand()
    daos.env = EnvironmentVariables({"key": "value"})
    daos.pool_set_attr(pool, attr, value, svc)
    daos.pool_list_attrs(pool, svc)
```
The current `pool_query()`, `container_create()`, and `container_destroy()` `DaosCommand` methods do not pass in `env`.  The `pool_list_cont`, `pool_set_attr`, `pool_get_attr`, and `pool_list_attrs` should have their `env` argument removed.

```suggestion 
    def pool_set_attr(self, pool, attr, value, svc):
```

Reply:
I've been wondering about this env. Thanks for the explanation.